config/routes.rb

```diff
@@ -63,6 +63,7 @@
 
   # catch all mailer preview paths
   get '/rails/mailers/*path' => 'rails/mailers#preview'
-  # required for Angular html5mode
-  get '*path' => 'home#index'
+
+  # catch all to send all HTML requests to Angular (html5mode)
+  get '*path', to: 'home#index', constraints: -> (req) { req.format == :html }
```

Comment:
The other way didn't fully work 😊 because it only sent ".html" requests to angular. 

So I was playing around with this 404 behavior a bit and it works great for the most part the way Rails handles it out of the box -- the only thing that is perhaps a little annoying is that in the logs it still shows a RouterError and a stack trace (but the fact that it's serving a 404 and not a 500 should fix the "false" errors in NewRelic at least). 

So I looked into a couple alternatives to handle it if we wanted cleaner logging:
- Final catch all route that goes to an error handler controller which serves up a 404. Pro: no error w/ stack trace. Con: adding extra setup e.g. responding in the correct format that is basically replicating what Rails seems to be doing for you in its prebuilt 404 behavior. 
- Logger initializer that captures RouteErrors and just logs them differently / without stack trace. Also feels like somewhat unnecessary/ overconfiguration but I like it better as an option. 
- or just leave it alone?